lib/platform/azure/index.ts

```diff
@@ -35,6 +35,7 @@ import { smartTruncate } from '../utils/pr-body';
 import * as azureApi from './azure-got-wrapper';
 import * as azureHelper from './azure-helper';
 import type { AzurePr } from './types';
+import { AzurePrVote } from './types';
```

Comment:
Merge with lone before and remove `type`

Reply:
I have merged the type import statements.